### ISSUE TITLE
If a theme is applied, use the theme's token in the `Avatar`, `AvatarGroup` and `AvatarBadge` component

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1087]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.0

### Description

Currently, styling values in components ignore theme tokens. If a theme is applied, you should use the theme's token.

Yamada UI's style system provides fallback functions. If the value of the token in the first argument does not exist, it returns the second argument, the fallback value.

```diff
const {
  className,
  src,
  srcSet,
  name,
  loading,
  alt,
  icon,
  ignoreFallback,
  referrerPolicy,
-  borderRadius = "9999px",
+  borderRadius = "fallback(full, 9999px)",
-  rounded = "9999px",
+  rounded = "fallback(full, 9999px)",
  onError,
  onLoad,
  crossOrigin,
  format,
  children,
  ...rest
} = omitThemeProps(mergedProps)
```

```diff
const {
  className,
  max,
  borderColor,
-  gap = "-0.5rem",
+  gap = "fallback(-2, -0.5rem)",
-  borderRadius = "9999px",
+  borderRadius = "fallback(full, 9999px)",
-  rounded = "9999px",
+  rounded = "fallback(full, 9999px)",
  children,
  ...rest
} = omitThemeProps(props)
```

```diff
{ping ? (
  <ui.div
    className="ui-avatar__badge__ping"
    __css={{
      position: "absolute",
      boxSize: "100%",
-      rounded: "9999px",
+      rounded: "fallback(full, 9999px)",
      opacity: 0.75,
      zIndex: -1,
      bg: pingColor,
    }}
    animation={animation}
  />
) : null}
```

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

**Comment by Melonps** [comment]
Hi! May I take on the challenge of this issue? 
I've actually gone this [article](https://zenn.dev/hirotomoyamada/articles/711e4727d7a56e)

**Comment by taroj1205** [comment]
@Melonps 

Hi, thanks for the comment!

Yes, please.

I'll assign it to you! Good luck!

**Comment by hirotomoyamada** [comment]
Fixed #1149
